### PR TITLE
Report on Cuba internet censorship

### PR DESCRIPTION
**Pull request by agrabeli**
This report is an OONI research study on internet censorship in Cuba.

### CODE REVIEW

File: content/post/cuba-internet-censorship-2017.md [hunk 1]
```diff
@@ -0,0 +1,1923 @@
...
+as “[El Paquete Semanal](http://www.cbc.ca/news/world/cuba-el-paquete-internet-wifi-havana-1.3527274)”
+(“The Weekly Package”), or “El Paquete” for short.
+
+El Paquete consists of a [variety of digital materials](https://laredcubana.blogspot.it/2016/05/inside-edition-of-el-paquete.html),
+ranging from music, TV series, movies, and video clips, to news
+websites, software manuals, classifieds, and advertisements. Every week,
+the El Paquete package contains different materials. Cubans can purchase
+these packages by having the materials copied onto CDs or USB drives.
+
+Interestingly enough, these materials [do not include pornography or any other prohibited materials](http://www.cbc.ca/news/world/cuba-el-paquete-internet-wifi-havana-1.3527274),
```
Comment (willscott) [inline content/post/cuba-internet-censorship-2017.md]: -"interestingly enough"

"These materials [do not include pornography or other prohibited materials], nor do they include content expressing criticism towards the Cuban government.

File: content/post/cuba-internet-censorship-2017.md [hunk 2]
```diff
@@ -0,0 +1,1923 @@
...
+government. This has led some to
+[speculate](http://www.cbc.ca/news/world/cuba-el-paquete-internet-wifi-havana-1.3527274)
+that the Cuban government might be involved in its production.
+
+In a way, El Paquete emerged as a sort of substitute for broadband
+internet, serving as Cuba’s “[offline internet](https://www.theguardian.com/world/2014/dec/23/cuba-offline-internet-weekly-packet-external-hard-drives)”.
+It enables Cubans to gain access to online content, without being
+online.
+
+While in Cuba, we asked where we could go to buy the weekly El Paquete.
```
Comment (willscott) [inline content/post/cuba-internet-censorship-2017.md]: In at least some neighborhoods of havana, el paquete was advertised in storefronts, so I'm not sure this paragraph is useful in communicating the level of 'underground'-ness of the distribution.

**Comment by hellais** [comment]
Made some minor edits and merged.

File: content/post/cuba-internet-censorship-2017.md [hunk 3]
```diff
@@ -0,0 +1,1923 @@
...
+measurements, was found to reset connections to those sites and serve
+(blank) block pages. Only the HTTP version of those sites was blocked,
+potentially enabling users to circumvent the censorship by merely
+accessing them over HTTPS.
+
+Most of the blocked sites express criticism towards Cuba’s government,
+either directly or indirectly. While various web proxies were found to
+be blocked, it’s worth noting that the [Tor network](https://www.torproject.org/) was found to be
+[accessible](https://explorer.ooni.torproject.org/measurement/20170607T175300Z_AS27725_XXfB14TYx5szrWoFiokwjWdQZIJQIuEtGWpkfaojBEPdiNsc3u)
+across Cuba. Similarly, popular communications tools, like [Facebook Messenger](https://explorer.ooni.to
```
Comment (willscott) [inline content/post/cuba-internet-censorship-2017.md]: what about revolico? http://www.reuters.com/article/us-cuba-internet-idUSTRE5A013V20091101

Comment (agrabeli) [reply content/post/cuba-internet-censorship-2017.md]: We actually found revolico to be accessible: https://explorer.ooni.torproject.org/measurement/20170601T235618Z_AS27725_SpZcfWCu3e17GPv8jpXSOIQucnRNNPolNnG2OODcGQC8H70cbe?input=http:%2F%2Fwww.revolico.com%2F

File: content/post/cuba-internet-censorship-2017.md (hunk 2 above)
Comment (agrabeli) [reply content/post/cuba-internet-censorship-2017.md]: Ah, didn't know. I'll delete that paragraph in that case. :)

File: content/post/cuba-internet-censorship-2017.md [hunk 4]
```diff
@@ -0,0 +1,1923 @@
...
+not always provide fast and reliable internet, can in itself be viewed
+as a barrier to accessing the internet in Cuba.
+
+Since [most of the wifi hotspots are located in parks](http://www.etecsa.cu/internet_conectividad/areas_wifi/), we
+dubbed them “*ParkNets*”. When hanging out in ParkNets, we noticed that
+almost all Cubans accessed the internet via their mobile phones. We also
+noticed that they appeared to be using the internet primarily for
+communications. When talking to locals, they mentioned that - in their
+view, at least - most Cubans use the internet to connect with their
+family and friends (many of whom are abroad), and that [Facebook Messenger is extremely popular](https://www.economist.com/news/americas/21716099-ingenious-answer-digital-deprivation-cuba-app-stores-pay-rent).
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/post/cuba-internet-censorship-2017.md]: Good point, I'll add it.

File: content/post/cuba-internet-censorship-2017.md [hunk 5]
```diff
@@ -0,0 +1,1923 @@
...
+Cubans therefore have a uniquely different relationship with the
+internet, in comparison to other countries. They don’t access the
+internet, they *visit* it.
+
+Currently, Cuba has [421 public wifi hotspots](http://www.etecsa.cu/internet_conectividad/areas_wifi/). The
+vast majority of these hotspots are located in public parks. Others are
+located in cafeterias, boulevards, and research centres. One of the wifi
+hotspots in Santa Clara is, interestingly enough, located right under
+the Che Guevara Mausoleum. [Most wifi hotspots (63) are located in Havana](http://www.etecsa.cu/internet_conectividad/areas_wifi/), the
+country’s capital. Even though Santiago de Cuba is the country’s second
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/post/cuba-internet-censorship-2017.md]: Possibly... the criteria behind the distribution of hotspots are not entirely clear to me...

File: content/post/cuba-internet-censorship-2017.md [hunk 6]
```diff
@@ -0,0 +1,1923 @@
...
+
+ETECSA provides [ADSL services](http://www.etecsa.cu/internet_conectividad/internet/#adsl)
+for the commercial and state sector. Given that only 1 Mbps costs 16,000
+CUC per year, it’s clear that ADSL services are currently out of reach
+for most Cubans. ETECSA also offers [point-to-point connections](http://www.etecsa.cu/internet_conectividad/conectividad/#puntoapunto)
+to businesses at a fixed monthly rate. According to ETECSA, full-time
+connection is guaranteed with a maximum transmission speed of up to 2
+Mbps. Similarly, ETECSA enables businesses to access and use the
+internet via [ATM and Ethernet](http://www.etecsa.cu/internet_conectividad/conectividad/#ATM).
+All such services, however, cannot be used by the Cuban public, mainly
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/post/cuba-internet-censorship-2017.md]: Thanks, it's worth adding that there.

**Comment by agrabeli** [comment]
@willscott thanks so much for your feedback! I'll edit accordingly.

File: content/post/cuba-internet-censorship-2017.md [hunk 7]
```diff
@@ -0,0 +1,1923 @@
...
+blocked, the [Tor network](https://www.torproject.org/) was
+[accessible](https://explorer.ooni.torproject.org/measurement/20170607T175300Z_AS27725_XXfB14TYx5szrWoFiokwjWdQZIJQIuEtGWpkfaojBEPdiNsc3u)
+during the testing period of this study.
+
+Chinese vendor Huawei was found to be supporting Cuba’s internet
+infrastructure. While it is clear that Cuba is using Huawei’s access
+points, it remains unclear whether and to what extent Huawei equipment
+is actually being used to implement internet censorship in the country.
+
+It was clear though that Google was blocking IP addresses originating
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/post/cuba-internet-censorship-2017.md]: Thanks, addressed.

File: content/post/cuba-internet-censorship-2017.md [hunk 8]
```diff
@@ -0,0 +1,1923 @@
...
+It’s worth noting that unless users have deleted all of their browser
+cookies, they are unable to login via the Nauta portal. This in itself
+could potentially be viewed as a barrier to accessing the internet for
+users who don’t know how to delete their browser cookies (or who aren’t
+aware that they should be doing so).
+
+Permanent accounts don’t require ETECSA cards. Rather, Cubans are issued
+usernames and passwords when they purchase permanent accounts from
+ETECSA, which they can charge and recharge for up to a year at a time.
+Interestingly enough, Cubans can top up their permanent accounts to
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/post/cuba-internet-censorship-2017.md]: Thanks, addressed.

File: content/post/cuba-internet-censorship-2017.md [hunk 9]
```diff
@@ -0,0 +1,1923 @@
...
+option for temporary accounts, perhaps because it is assumed that they
+are mostly used by tourists. In addition to internet and/or intranet
+access, permanent accounts also provide Cubans with an @nauta.cu email
+address.
+
+![ETECSA accounts](/post/cuba-internet-censorship-2017/cu-05.png)
+
+Source: ETECSA, http://www.etecsa.cu/internet\_conectividad/internet/
+
+When we visited ETECSA offices in Cuba to purchase temporary accounts,
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/post/cuba-internet-censorship-2017.md]: Thanks, I've replaced that paragraph with the one you're suggesting.

File: content/post/cuba-internet-censorship-2017.md (hunk 1 above)
Comment (agrabeli) [reply content/post/cuba-internet-censorship-2017.md]: Thanks, edited.

**Pull request merged**